cmd/config/config.go

```diff
@@ -286,6 +286,15 @@ func ReadMountSpec(r *types.RunConfig, flags *pflag.FlagSet) (*types.MountSpec,
 }
 
 func applyKernelCmdline(r *types.RunConfig, mount *types.MountSpec) error {
+	lsm, err := r.Config.Fs.ReadFile("/sys/kernel/security/lsm")
```

Comment:
SELinux could set by default without needing any kernel param, hence I think is is just a safer test to check if SELinux is enabled within this kernel execution or not.

Reply:
Looks good! I would move this out of the `applyKernelCmdline` method tho..